scene-lib/src/scenelib/annotations/el/LocalLocation.java

```diff
@@ -1,91 +1,219 @@
 package scenelib.annotations.el;
 
+import org.objectweb.asm.Label;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
 import java.util.Objects;
 
+import org.checkerframework.checker.nullness.qual.Nullable;
+
 /**
  * A {@link LocalLocation} holds location information for a local
- * variable: slot index, scope start, and scope length.
+ * variable: slot index, scope start, and scope end.
```

Comment:
This class's implementation represents a list of (slot index, scope start, scope end) triples.  However, the class documentation does not document it that way.  Should it?